idaes/core/components.py

```diff
@@ -54,18 +54,27 @@ class ComponentData(ProcessBlockData):
 
     CONFIG.declare("dens_mol_liq_comp", ConfigValue(
         description="Method to use to calculate liquid phase molar density"))
+    CONFIG.declare("dens_mol_sol_comp", ConfigValue(
+        description="Method to use to calculate solid phase molar density"))
+
     CONFIG.declare("cp_mol_liq_comp", ConfigValue(
         description="Method to calculate liquid component specific heats"))
+    CONFIG.declare("cp_mol_sol_comp", ConfigValue(
```

Comment:
@andrewlee94 Should there also be a test for this? I noticed that you added tests for the other properties that you introduced, with the exception of `cp_mol_sol_comp`. 

I was thinking this would just need modification in `test_cv_mol_phase_comp` in `test_ideal.py`.

Reply:
There really isn't much need to test config arguments (especially if you don't set a default or domain). If it is missing, it should show up in a later test that tried to use the argument.

Reply:
Ah, OK. I guess I just thought you'd want to add the corresponding lines for `sol` the same way you did for `liq` and `vap` (lines 183-191 in `test_ideal.py`):

```
m.params.get_component(j).config.cp_mol_liq_comp = dummy_call
m.params.get_component(j).config.cp_mol_ig_comp = dummy_call       
 
assert str(Ideal.cv_mol_phase_comp(m.props[1], "Liq", j)) == str(42)        
assert str(Ideal.cv_mol_phase_comp(m.props[1], "Vap", j)) == str(42 - pyunits.convert(const.gas_constant,                to_units=pyunits.kg*pyunits.m**2/pyunits.s**2 /pyunits.mol/pyunits.K))
```